<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>691</width>
      <height>306</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Name">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Phone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Phone">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_Save">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Clear">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ClearOne">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Clear One</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionpen"/>
    <addaction name="actionclear"/>
    <addaction name="actionsave"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionnew"/>
   <addaction name="actionpen"/>
   <addaction name="actionclear"/>
   <addaction name="actionsave"/>
   <addaction name="actionquit"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionnew">
   <property name="icon">
    <iconset resource="resrc.qrc">
     <normaloff>:/Images/new.png</normaloff>:/Images/new.png</iconset>
   </property>
   <property name="text">
    <string>new</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ShortcutContext::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionpen">
   <property name="icon">
    <iconset resource="resrc.qrc">
     <normaloff>:/Images/open.png</normaloff>:/Images/open.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionclear">
   <property name="icon">
    <iconset resource="resrc.qrc">
     <normaloff>:/Images/clear.png</normaloff>:/Images/clear.png</iconset>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="icon">
    <iconset resource="resrc.qrc">
     <normaloff>:/Images/about.png</normaloff>:/Images/about.png</iconset>
   </property>
   <property name="text">
    <string>about</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="resrc.qrc">
     <normaloff>:/Images/save.png</normaloff>:/Images/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="icon">
    <iconset resource="resrc.qrc">
     <normaloff>:/Images/exit.png</normaloff>:/Images/exit.png</iconset>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resrc.qrc"/>
 </resources>
 <connections/>
</ui>
